.ci.yaml

```diff
@@ -4027,7 +4027,85 @@ targets:
         [
           {"dependency": "vs_build", "version": "version:vs2019"}
         ]
-      task_name: hello_world_windows__compile
+      task_name: hello_world_win_desktop__compile
+
+  - name: Windows flutter_gallery_win_desktop__compile
+    recipe: devicelab/devicelab_drone
+    bringup: true
```

Comment:
Can you add a TODO with the tracking bug for ensuring these get moved off bringup?

Reply:
Could you clarify what tracking bug you are referring to and how it is related to the bringup status?

Reply:
@CaseyHillers I thought the infra automatically removed the bringup tag after some number of successful runs (50?). I was under the impression that the TODO tracking bug was just for flakes. Is that incorrect? 

I can see the case for wanting it for new tests too, but wasn't aware this was policy, and hadn't been doing it for any I'd added. If it's required, we should probably document that in the [bringup example](https://github.com/flutter/cocoon/blob/9eeb5f26977fce5a98d512c94a69149f2fdeed16/CI_YAML.md?plain=1#L55-L62) in CI_YAML.md.

Reply:
The tracking bug is a good to have for new targets, but is not officially enforced. As pointed out, it is now only enforced for ones from flake bot.

Another context: if you add a tracking bug here, the bot only removes the `bringup: true` when the bug is closed. If the tracking bug is not intended to close in a short period, we need to leave it empty. Otherwise the target will remain as `bringup: true`.

Reply:
I do not see any other TODO comments in this file yet. Is there a particular format it needs to be in for the bot to recognize it?